business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/RepositoryIntegrationTest.java

```diff
@@ -42,6 +43,7 @@ public static void createSpace() {
         createSpace(SPACE);
     }
 
+    @Ignore
```

Comment:
please add here a reason and jira URL you created for fixing the tests in future